website/docs/developer/development-guide/code/versioning-releases.md

```diff
@@ -150,6 +150,9 @@ After hitting the play button on `Deploy DR ORCA Buckets` stage in bamboo plan,
 - PREFIX
 - DR_AWS_ACCESS_KEY_ID
 - DR_AWS_SECRET_ACCESS_KEY
+- CUMULUS_ACCOUNT_ID

```

Comment:
See notes in `bamboo-specs/specs/deploy-cumulus-orca-buckets.yaml` on removing this as a variable that users will have to input a value into.

Reply:
removed